.pre-commit-config.yaml

```
@@ -1,45 +1,81 @@
+exclude: ^\.pixi
 repos:
-  - repo: https://github.com/Quantco/pre-commit-mirrors-insert-license
-    rev: 1.3.0
+  - repo: local
     hooks:
+      # ensure pixi environments are up to date
+      # workaround for https://github.com/prefix-dev/pixi/issues/1482
+      - id: pixi-install
+        name: pixi-install
+        entry: pixi install -e default -e lint
+        language: system
+        always_run: true
+        require_serial: true
+        pass_filenames: false
       - id: insert-license
+        name: insert-license
+        entry: pixi run -e lint insert-license
         types: [python]
+        language: system
         args:
           - --license-base64
           - IyBDb3B5cmlnaHQgKGMpIFF1YW50Q28gMjAyNC0yMDI0CiMgU1BEWC1MaWNlbnNlLUlkZW50aWZpZXI6IEJTRC0zLUNsYXVzZQo=
```

Please replace the base64 with 

```suggestion
          - Q29weXJpZ2h0IChjKSBRdWFudENvIDIwMjQtMjAyNApTUERYLUxpY2Vuc2UtSWRlbnRpZmllcjogQlNELTMtQ2xhdXNl
```